homeassistant/components/almond/config_flow.py

```diff
@@ -73,12 +77,15 @@ async def async_oauth_create_entry(self, data: dict) -> dict:
         data["host"] = self.host
         return self.async_create_entry(title=self.flow_impl.name, data=data)
 
-    async def async_step_import(self, user_input: dict = None) -> dict:
+    async def async_step_import(
+        self, user_input: dict[str, Any] | None = None
```

Comment:
When can the user input be None? Normally the import flow always receives some existing data as input.

Reply:
OK, that makes sense.
It is called like that by some tests; we should probably change the tests instead.